exporter/stackdriverexporter/factory_test.go

```diff
@@ -20,12 +20,15 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"go.uber.org/zap"
+
+	"github.com/open-telemetry/opentelemetry-collector/config/configcheck"
```

Comment:
We are not being consistent with the grouping between different files, creates https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/28 to track it.